js/tracker.php

```diff
@@ -22,9 +22,19 @@
  *
  * @see core/Piwik.php
  */
-define('PIWIK_INCLUDE_PATH', '..');
 define('PIWIK_DOCUMENT_ROOT', '..');
-define('PIWIK_USER_PATH', '..');
+
+if (file_exists(PIWIK_DOCUMENT_ROOT . '/bootstrap.php')) {
+    require_once PIWIK_DOCUMENT_ROOT . '/bootstrap.php';
+}
+
+if (!defined('PIWIK_INCLUDE_PATH')) {
```

Comment:
same behaviour now as in `index.php` as `bootstrap.php` could change the user path